src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -145,7 +145,10 @@ private function parseLongOption($token)
         $name = substr($token, 2);
 
         if (false !== $pos = strpos($name, '=')) {
-            $this->addLongOption(substr($name, 0, $pos), substr($name, $pos + 1));
+            if (0 === strlen($value = substr($name, $pos + 1))) {
```

Comment:
Looks fine. See #20008
Btw, could you take a look on [this check](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Input/ArgvInput.php#L225)? I think we can do the same